t/50throttle-response.t

```diff
@@ -0,0 +1,41 @@
+use strict;
+use warnings;
+use Test::More;
+use t::Util;
+
+plan skip_all => 'curl not found'
+  unless prog_exists('curl');
+
+my $all_data = do {
+  open my $fh, "<", "@{[DOC_ROOT]}/halfdome.jpg"
+    or die "failed to open file:@{[DOC_ROOT]}/halfdome.jpg:$!";
+  undef $/;
+  <$fh>;
+};
+
+my $server = spawn_h2o(<< "EOT");
+throttle-response: ON
+hosts:
+  default:
+    paths:
+      /:
+        file.dir: @{[ DOC_ROOT ]}
+        header.add: "X-Traffic: 10000"
+EOT
+
+run_with_curl($server, sub {
+                my ($proto, $port, $curl_cmd) = @_;
+                $curl_cmd .= " --silent --show-error";
+                my $url = "$proto://127.0.0.1:$port/halfdome.jpg";
+
+                subtest "throttle-to-low-speed" => sub {
+                  my $start_time = time;
+                  my $resp = `$curl_cmd $url`;
+                  my $end_time = time;
```

Comment:
Please add `use Time::HiRes qw(time);` to the top of the file so that `time` would return a floating point number.

With the change, I think you can increase the value of `x-traffic` to somewhere around 100000 to save time spent in running the test, thanks to more accurate measurement.